tests/src/mods/constants.rs

```diff
@@ -0,0 +1,85 @@
+// Wasm names
+pub const CONTRACT_KYC: &str = "kyc-contract.wasm";
+pub const CONTRACT_CEP_47_TOKEN: &str = "cep_47.wasm";
+pub const CONTRACT_CEP_78_TOKEN: &str = "cep_78.wasm";
+pub const CONTRACT_AUCTION: &str = "casper-private-auction-installer.wasm";
+pub const SESSION_BID_PURSE: &str = "bid-purse.wasm";
+pub const SESSION_EXTENDED_BID_PURSE: &str = "extend-bid-purse.wasm";
+pub const SESSION_DELTA_BID_PURSE: &str = "delta-bid-purse.wasm";
+
+// Named Keys
+pub const KEY_CEP_47_CONTRACT_NAME: &str = "cep_47_token";
+pub const KEY_CEP_47_CONTRACT_HASH: &str = "cep_47_token_contract_hash";
+pub const KEY_CEP_47_PACKAGE_HASH: &str = "cep_47_token_package_hash";
+
+pub const KEY_CEP_78_CONTRACT_HASH: &str = "nft_contract";
```

Comment:
[⚠️](https://emojiguide.com/symbols/warning/#:~:text=warning%20%E2%9A%A0%EF%B8%8F,something%20that%20might%20harm%20them.) It seems that this specific name and prefix is`nft_contract` and below `nft_contract_package` can not be set when creating a minting contract with CEP-78 while the name of the contract could be set in CEP-47.

Reply:
Correct, already ticketed to be changed but there is debate on it. Feel free to bring it up on meeting.